packages/theme-a11y/theme-a11y.test.js

```diff
@@ -135,6 +135,11 @@ describe('bindInPageLinks()', () => {
 
     expect(Array.isArray(links)).toBeTruthy;
     expect(links.length).toBe(1);
+    expect(links).toEqual(
+      expect.not.arrayContaining(
+        Array.from(document.querySelectorAll('.js-ignore-link'))
```

Comment:
If you use `.querySelector()` you can ditch the `Array.from()` 😄

Reply:
But `.querySelector()` will only return the first matching element - would make more sense to `.querySelectorAll` to match the real behaviour? What do you think?

Reply:
True!

On another note -- the [examples I'm seeing of arrayContaining](https://jestjs.io/docs/en/expect#expectarraycontainingarray) put the `.not()` with `.not.toEqual()` -- can you make sure this test fails if you comment out the ignore portion the function?

Reply:
[expect.not.arrayContaining](https://jestjs.io/docs/en/expect#expectnotarraycontainingarray) is good 👍 tested both positive and negative to make sure everything is working as expected